CONTRIBUTING.md

```diff
@@ -0,0 +1,4 @@
+# Contributing to Eclipse Platform
+See https://github.com/eclipse-platform/.github/blob/main/CONTRIBUTING.md for contributing to Eclipse Platform in general.
+
+Additional SWT-specific information can be found in [README.md](README.md).
```

Comment:
I'd propose to move the information related to making contributions from `README.md` to `CONTRIBUTING.MD` and have a `How to Contribute` section in the `README.md` referring to the `CONTRIBUTING.MD`. 
Would make it consistent across repositories.